getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,73 @@
+# Instructions for macOS
+
+Install [Homebrew](http://brew.sh/) to give your mac a package manager. Open up your terminal emulator of choice and run:
+
+```shell
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
```

Comment:
Is this needed? If it is, the user will either need to us the GNU version of the command explicitly (i.e. `grm` instead of `rm`), or change their path:
```shell
# If homebrew is installed, set PATH to default to GNU tools
if [ -f /usr/local/bin/brew ]; then
        export PATH=/usr/local/opt/coreutils/libexec/gnubin:/usr/local/sbin:$PATH
        export MANPATH=/usr/local/opt/coreutils/libexec/gnuman:$MANPATH
fi
``